frontend/packages/ceph-storage-plugin/src/components/block-pool/block-pool-list-page.tsx

```diff
@@ -0,0 +1,293 @@
+      transforms: [sortable],
+      props: { className: blockPoolColumnInfo.name.classes },
+    },
+    {
+      title: blockPoolColumnInfo.status.title,
+      id: blockPoolColumnInfo.status.id,
+      props: { className: blockPoolColumnInfo.status.classes },
+    },
+    {
+      title: blockPoolColumnInfo.storageclasses.title,
+      id: blockPoolColumnInfo.storageclasses.id,
+      props: { className: blockPoolColumnInfo.storageclasses.classes },
+    },
+    {
+      title: blockPoolColumnInfo.replicas.title,
+      id: blockPoolColumnInfo.replicas.id,
+      props: { className: blockPoolColumnInfo.replicas.classes },
+    },
+    {
+      title: blockPoolColumnInfo.usedcapacity.title,
+      id: blockPoolColumnInfo.usedcapacity.id,
+      transforms: [wrappable],
+      props: { className: blockPoolColumnInfo.usedcapacity.classes },
+    },
+    {
+      title: blockPoolColumnInfo.mirroringstatus.title,
+      id: blockPoolColumnInfo.mirroringstatus.id,
+      transforms: [wrappable],
+      props: { className: blockPoolColumnInfo.mirroringstatus.classes },
+    },
+    {
+      title: blockPoolColumnInfo.overallImagehealth.title,
+      id: blockPoolColumnInfo.overallImagehealth.id,
+      transforms: [wrappable],
+      props: { className: blockPoolColumnInfo.overallImagehealth.classes },
+    },
+    {
+      title: blockPoolColumnInfo.compressionstatus.title,
+      id: blockPoolColumnInfo.compressionstatus.id,
+      transforms: [wrappable],
+      props: { className: blockPoolColumnInfo.compressionstatus.classes },
+    },
+    {
+      title: blockPoolColumnInfo.compressionsavings.title,
+      id: blockPoolColumnInfo.compressionsavings.id,
+      transforms: [wrappable],
+      props: { className: blockPoolColumnInfo.compressionsavings.classes },
+    },
+    {
+      title: '',
+      props: { className: Kebab.columnClass },
+    },
+  ];
+};
+
+const getRows: RowFunction<K8sResourceKind> = (props) => <BlockPoolTableRow {...props} />;
+
+const BlockPoolTableRow: React.FC<RowFunctionArgs> = ({ obj, index, key, style, customData }) => {
+  const { t } = useTranslation();
+  const blockPoolColumnInfo = BlockPoolColumnInfo(t);
+  const props: BlockPoolListRowProps = customData;
+  const { name, namespace } = obj.metadata;
+  const replica: string = obj.spec?.replicated?.size;
+  const mirroringStatus: boolean = obj.spec?.mirroring?.enabled;
+  const mirroringImageHealth: string = mirroringStatus
+    ? obj.status?.mirroringStatus?.summary?.image_health
+    : '-';
+  const lastChecked: string = obj.status?.mirroringStatus?.lastChecked;
+  const formatedDateTime = lastChecked
+    ? twelveHoursdateTimeNoYear.format(new Date(lastChecked))
+    : '-';
+  const compressionStatus: boolean = obj.spec?.compressionMode === COMPRESSION_ON;
+  const phase = obj?.status?.phase;
+
+  // Hooks
+  const poolScNames: string[] = React.useMemo(
+    () => getScNamesUsingPool(props?.storageClasses, name),
+    [name, props],
+  );
```

Comment:
```suggestion
  const poolScNames: string[] = React.useMemo(
    () => getScNamesUsingPool(props?.storageClasses, name),
    [name, props?.storageClass],
  );
```

Reply:
ack

Reply:
I tried props?.storageClass when commit lint check is removing this change and keeping props only